src/BSON/BinaryInterface.stub.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+  * @generate-class-entries static
+  * @generate-function-entries
```

Comment:
Note that for interfaces there is no `static` suffix behind `generate-function-entries`. Specifying that generates an invalid stub file due to a bug in gen_stub.php. I'll look at that separately.

Reply:
What does `static` do for `generate-class-entries`?

Reply:
Yes, the tag value is injected into the function definitions: `generate-class-entries` maps to the `register_class_MongoDB_...` function, while `generate-function-entries` maps to the `ZEND_METHOD(MongoDB_..., ...)` lines.